Tests/WeatherSDKTests/GetCityAnnouncementsErrorTest.swift

```diff
@@ -1,6 +1,7 @@
 // Code generated by smithy-swift-codegen. DO NOT EDIT!
 
 import ClientRuntime
+import SmithyReadWrite
```

Comment:
`Tests/WeatherSDKTests` contains auto-generated protocol tests for WeatherSDK.  Updated for new codegen.